packages/block-editor/src/components/use-moving-animation/index.js

```diff
@@ -144,18 +145,28 @@ function useMovingAnimation(
 	return prefersReducedMotion
 		? {}
 		: {
-				transformOrigin: 'center',
-				transform: interpolate(
-					[ animationProps.x, animationProps.y ],
-					( x, y ) =>
-						x === 0 && y === 0
-							? undefined
-							: `translate3d(${ x }px,${ y }px,0)`
-				),
+				left: interpolate( [ animationProps.x ], ( x ) => {
+					x = Math.round( x );
+
+					if ( x === 0 ) {
+						return;
+					}
+
+					return `${ x }px`;
+				} ),
+				top: interpolate( [ animationProps.y ], ( y ) => {
+					y = Math.round( y );
+
+					if ( y === 0 ) {
+						return;
+					}
+
+					return `${ y }px`;
+				} ),
```

Comment:
What happens if the block is left/right aligned or potentially absolute positioned... That's my main concern,  it seems that the "transform" trick works no matter how the block is positioned while the "left/top" depends on the positioning of the block? It might not be a big issue because we have "wrappers" for aligned blocks but I wonder if there's third-party absolute positioned block this change could break?

Reply:
Let me see if it works fine with a transform. I started with trying top/left because some browser automatically round this, not not all. We should be able to just round thee value and use transform.

Reply:
I changed it. Should be the same